libs/shared/ui/src/styles/globals.css

```
@@ -72,13 +74,12 @@
-
-@layer base {
-  * {
-    @apply border-border;
+  a {
+    transition: all 0.2s ease-in-out;
   }
-  body {
-    @apply bg-background text-foreground;
+
+  a:hover {
+    color: theme('colors.link.hover');
```

Biome (1.9.4)

[error] Unexpected unknown function: theme

Use a known function instead.
See MDN web docs for more details.

(lint/correctness/noUnknownFunction)